src/modules/checkout/server/procedures.ts

```
@@ -11,8 +11,55 @@ import {
   createTRPCRouter,
   protectedProcedure
 } from "@/trpc/init";
+import { te } from "date-fns/locale";
```

ESLint

[error] 'te' is defined but never used.

(@typescript-eslint/no-unused-vars)